### ISSUE TITLE
Gutenberg no longer shows previews while editing

### ISSUE THREAD

**Issue by d-damba** [WordPress/gutenberg#50491]
### Description

This just stopped working.  No plugin changes, wordpress changes etc. Just stopped working and no longer shows changes being made in the editor. They show on the front end, just not in gutenberg.

### Step-by-step reproduction instructions

Unknown

### Screenshots, screen recording, code snippet

![heading1](https://github.com/WordPress/gutenberg/assets/10174/8f4a5f2b-41da-499e-ad83-36e48f5347ee)
![heading5](https://github.com/WordPress/gutenberg/assets/10174/91d4074f-92ba-43bc-a407-ae5636c9c313)

### Environment info

Wordpress 6.1.1

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ramonjd** [comment]
I've just tested on the latest Gutenberg (fresh install, no plugins activated) and I can see all my changes in the editor.

Which version of Gutenberg do you have installed? It looks like the Yoast plugin is activated on the screenshot. Could there be other plugins that are activated?

If it's a plugin conflict issue, it might be worth asking in the WordPress support forums: https://wordpress.org/support/forums/

Also if there is more info on how to reproduce it can be placed under the "Step-by-step reproduction instructions" heading.

**Comment by d-damba** [comment]
Hi, that's the strange thing. Nothing changed on the site as far as plugins or updates to wp. I'm not exactly sure of the Gutenberg version but I assume it's 14.1 since it's Wordpress 6.1.1.

I've tried disabling plugins, including Yoast, but no go.  It seems to be mostly just the titles and headings, as far as I can tell, others elements will preview in the editor fine (Lists, columns, images, etc.)

I wish I had more for step by step, but I'm not sure. It was working, nothing was changed on the backend of the site, then it stopped working. 

I've tried it on Firefox and Chrome, flushed the cache, no relevant errors in the console and it's the same for others editing the site.

I'm stumped.

**Comment by d-damba** [comment]
I just started working on this and took over as site admin. Whoever built the site installed all sorts of plugins, plenty of unnecessary ones. I've been spending days trying to figure out what is being used and what isn't. 

Anyway, it turns out that they installed two plugins called: MobiLoud News and MobiLoud Smart App Banners. Both of which were not being used. I deactivated both, it turns out for some weird reason, that was what caused it. It's got to be a combination of more, but I reactivated them for the hell of it and the editor is now showing previews again. Zero idea why at the moment.
